kpi/filters.py

```diff
@@ -206,7 +207,7 @@ def filter_queryset(self, request, queryset, view):
             q_obj = parse(
                 q, default_field_lookups=ASSET_SEARCH_DEFAULT_FIELD_LOOKUPS
             )
-        except ParseError:
+        except (ParseError, BadDefaultSearchException):
```

Comment:
We may want to make to a difference between `ParseError` and `BadDefaultSearchException` to return the correct message to users. 
So I think, we should catch the `BadDefaultSearchException` within the ViewSet.